blockly/vs-code-extension/src/handlers/sendBlocklyFile.ts

```diff
@@ -119,7 +120,7 @@ function displayErrorsInEditor(errorMessage: string, document: vscode.TextDocume
             currentError.symbol.type = '';
             currentError.errorOffset = -1;
 
-            currentLineNum = parseInt(errorMatch[1], 10) - 6; // Adjust for the wrapper code
+            currentLineNum = parseInt(errorMatch[1], 10) - wrapperOffset;
```

Comment:
jetzt "9", vorher "6"? absicht oder typo?

Reply:
9, sollte richtig sein. Ich habe 3 neue Zeilen in den Wrapper hinzugefügt über der `execute`-Methode. 
```java
...
import contrib.components.*;
import components.*;
import core.components.*;
```

Reply:
puh. das ist arg ... 

aber ok.